linux/LiveResponse/ClientAnalyzer/InstallXMDEClientAnalyzer.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/bash
```

Comment:
We might want to consider error handling for these. Also wget/unzip might not exist. So error handling would cover those as well.